src/memgraph.cpp

```diff
@@ -134,6 +134,9 @@ std::optional<Enum> StringToEnum(const auto &value, const auto &mappings) {
 }
 }  // namespace
 
+// Short help flag.
+DEFINE_bool(h, false, "Print usage and exit.");
```

Comment:
Since the macro creates a global variable we need this just to avoid the clang-tidy warning
```suggestion
// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
DEFINE_bool(h, false, "Print usage and exit.");
```